### PR TITLE
Display card

### PR DESCRIPTION
**Pull request by sidra2022**
Please review this component according to your availability.

### CODE REVIEW

File: packages/stencil/src/components/datacom-display-card/datacom-display-card-group.tsx
Comment (AngeCast) [inline packages/stencil/src/components/datacom-display-card/datacom-display-card-group.tsx:11]: If this is the only class being added AND it's always going to be applied, might as well just directly apply the class on line 15. You only really want to do this approach if you've got multiple conditional classes to render.

File: apps/storybook/src/stories/datacom-display-card.stories.tsx
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-display-card.stories.tsx:9]: Probably should have a space between in the title. (AKA 'Display Card')

File: apps/storybook/src/stories/datacom-display-card.stories.tsx
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-display-card.stories.tsx:18]: Remove empty line please.

File: apps/storybook/src/stories/datacom-display-card.stories.tsx
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-display-card.stories.tsx:53]: Code cleanliness thing. Please add a new empty line here to separate export from code above.

File: packages/stencil/src/components/datacom-display-card/datacom-display-card-group.css
Comment (AngeCast) [inline packages/stencil/src/components/datacom-display-card/datacom-display-card-group.css:15]: I'm assuming tablet should just have the cards one on top of each other? If so, that would be happening by default the way you've written, so you don't need to define the `grid-template-columns` OR the `grid-template-rows`

File: packages/stencil/src/components/datacom-display-card/datacom-display-card.tsx
Comment (AngeCast) [inline packages/stencil/src/components/datacom-display-card/datacom-display-card.tsx:21]: Probably best to use a tabindex of 0 here instead of a positive number. You might mess with the tab order when using keyboard navigation. (See below warning from the [Mozilla site](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex))

image

**Review by AngeCast — CHANGES_REQUESTED** [review]
Heya. Good stuff, but got a number of comments here and there. Have a look when you can.

File: apps/storybook/src/stories/datacom-display-card.stories.tsx
Comment (sidra2022) [reply apps/storybook/src/stories/datacom-display-card.stories.tsx:10]: Done.

File: apps/storybook/src/stories/datacom-display-card.stories.tsx
Comment (sidra2022) [reply apps/storybook/src/stories/datacom-display-card.stories.tsx]: Done.

**Review by AngeCast — DISMISSED** [review]
Changes seem fine now. Will merge AFTER the latest dev work has been merged into the main branch.

File: packages/stencil/src/components/datacom-display-card/datacom-display-card.css
Comment (AngeCast) [inline packages/stencil/src/components/datacom-display-card/datacom-display-card.css:106]: I believe this selector does nothing? You don't have a .dc-display-card-link class nested within another .dc-display-card-link class in your component HTML.

File: apps/storybook/src/stories/datacom-display-card.stories.tsx
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-display-card.stories.tsx:1]: Please move these stories into the "unstable" stories folder

**Review by AngeCast — CHANGES_REQUESTED** [review]

File: packages/stencil/src/components/datacom-display-card/datacom-display-card.tsx
Comment (AngeCast) [inline packages/stencil/src/components/datacom-display-card/datacom-display-card.tsx:31]: Probably don't need this empty div. Please see if you can remove it?

**Review by AngeCast — CHANGES_REQUESTED** [review]

File: packages/stencil/src/components.d.ts
Comment (AngeCast) [inline packages/stencil/src/components.d.ts:1]: Please delete this auto-generated file

**Review by AngeCast — CHANGES_REQUESTED** [review]

**Review by AngeCast — APPROVED** [review]

**Pull request merged**
